widgets/hierarchical-menu/hierarchical-menu.js

```diff
@@ -81,7 +81,7 @@ function hierarchicalMenu({
         templates
       });
 
-      cssClasses = {
```

Comment:
I'm not sure about this syntax, but here, aren't both `userCssClasses` and `cssClasses` defined when entering the function?